app.py

```diff
@@ -325,10 +328,21 @@ def open_curr_quest(quest_id):
     return render_template('curr_task_template.html', quest=quest)
 
 
+# Route to handle `Report Quest` Button
```

Comment:
That route should be in a separate file

Reply:
Last commit: 7d35494
- The route report_quest is moved to `edit_quest_form.py` file
- The imports are handled
- Everything works as expected